integration/framework.go

```diff
@@ -457,3 +458,21 @@ func (f *framework) ApplyAWSConfigPatch(patch []PatchSpec, clusterName string) e
 
 	return nil
 }
+
+func (f *framework) GetVersionBundleVersion(bundle []versionbundle.Bundle, vType string) string {
+	var output string
+	log.Printf("Tested version %q", vType)
+
+	for _, v := range bundle {
+		if (vType == "current" || vType == "") && !v.Deprecated && !v.WIP {
+			output = v.Version
+			break
+		}
```

Comment:
This is possible that we have multiple non-deprecated non-WIP versions. This isn't deterministic. I think you can use data to take latest.

Reply:
Added code to return the latest non-deprecated non-wip for current.